modules/node_groups/node_groups.tf

```diff
@@ -48,7 +48,7 @@ resource "aws_eks_node_group" "workers" {
   dynamic "launch_template" {
     for_each = each.value["launch_template_id"] == null && each.value["create_launch_template"] ? [{
       id      = aws_launch_template.workers[each.key].id
-      version = aws_launch_template.workers[each.key].latest_version
+      version = each.value["launch_template_version"] != null ? each.value["launch_template_version"] : aws_launch_template.workers[each.key].latest_version
```

Comment:
I was just wondering how user will get the `launch_template_version` if it's created directly by the module ? They have to go through the aws cli or aws console ?

Reply:
Yes I think they can see it in the TF or in cli/console. Most of the time if think you want to upgrade (that's one of the perk of MNG vs Self managed without instance refresh) but it might be useful when you have large pool and do not want to trigger an upgrade yet

Reply:
I just noticed that `launch_template_version = "$Latest"` by default https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/local.tf#L104. In that case, should we need this ternary evaluation ?

Does `version = each.value["launch_template_version"]` enough ?

Reply:
Yes it should be, fixed

Reply:
@ArchiFleKs @barryib 
Can we test this? Somehow "$Latest" is converted into the integer latest automatically in my experiments

https://github.com/terraform-aws-modules/terraform-aws-eks/issues/1446